tools/build.gradle

```diff
@@ -7,7 +7,7 @@ dependencies {
   compile project(':bmm') 
   compile project(':path-queries')
   compile project(':utils')
-  testCompile project(':openehr-rm')//TODO: this is temporary, this module should not depend on openehr-rm
+  compile project(':openehr-rm')//TODO: this is temporary, this module should not depend on openehr-rm
```

Comment:
And it is indeed. As discussed, the ModelInfoLookup should be passed to the constructor of RMObjectValidator, instead of depending on ArchieRMInfoLookup directly.